### PR TITLE
Add google style for docstrings #1784

### PR DESCRIPTION
**Pull request by katnoria**
This is my first PR to PySyft. 
In this PR, I have updated docstrings on two classes:

* frameworks/torch/tensors/abstract.py
* frameworks/torch/tensors/native.py

### CODE REVIEW

**Comment by iamtrask** [comment]
Looks like there are a few style errors - run black to fix :)

(pip install black)

then from the PySyft directory run

black syft

File: syft/frameworks/torch/tensors/abstract.py [hunk 1]
```diff
@@ -10,6 +10,11 @@ class AbstractTensor(ABC):
     """
 
     def wrap(self):
+        """Wraps the class inside torch tensor.
```
Comment (mari-linhares) [inline syft/frameworks/torch/tensors/abstract.py]: I think this comment could be more descriptive.

File: syft/frameworks/torch/tensors/abstract.py [hunk 2]
```diff
@@ -18,35 +23,42 @@ def wrap(self):
...
 
-            Returns:
-                binary: the serialized form of the tensor.
+        This is the high level convenience function for serializing torch
+        tensors. It includes three steps, Simplify, Serialize, and Compress as
+        described in serde.py.
 
-            Examples:
+        Args:
+            compress (bool): Whether or not to compress the object
```
Comment (mari-linhares) [inline syft/frameworks/torch/tensors/abstract.py]: Same.

File: syft/frameworks/torch/tensors/abstract.py [hunk 3]
```diff
@@ -18,35 +23,42 @@ def wrap(self):
...
     hook_self, cls, torch_tensor: bool = False, owner=None, id=None, *init_args, **init_kwargs
 ):
-
+    """Initializes the tensor.
+
+    Args:
+        hook_self: A reference to TorchHook class.
+        cls: An object to keep track of id, owner and whether it is a native
+            tensor or a wrapper over pytorch.
+        torch_tensor (bool): Whether or not it is torch tensor.
```
Comment (mari-linhares) [inline syft/frameworks/torch/tensors/abstract.py]: Same.

File: syft/frameworks/torch/tensors/native.py [hunk 4]
```diff
@@ -70,57 +67,52 @@ def create_pointer(
...
-                your pointer.
-
-            owner (....workers.BaseWorker): while "location" specifies where the
-                pointer points to, this parameter specifies the worker on which the
-                pointer is located. It is also where the pointer is registered if
-                register is set to True.
-
-            id (str or int): if you want to set the id of the pointer for
-                any special reason, you can set it here. Otherwise, it will be set
+            location (....workers.BaseWorker): The BaseWorker object which
```
Comment (mari-linhares) [inline syft/frameworks/torch/tensors/native.py]: Same for all these args.

File: syft/frameworks/torch/tensors/pointer.py [hunk 5]
```diff
@@ -43,34 +44,27 @@ def __init__(
...
-            "location" specifies where the pointer points to, this parameter
-            specifies the worker on which the pointer is located. It is also
-            where the pointer is registered if register is set to True.
-
-         * **id: ((str or int), optional)** - this is the id of the PointerTensor
-
+        """Initializes a PointerTensor.
+
+        Args:
+            parent (abstract.AbstractTensor): Optional, this is typically just
```
Comment (mari-linhares) [inline syft/frameworks/torch/tensors/pointer.py]: Same for all these args.

File: syft/frameworks/torch/tensors/pointer.py [hunk 6]
```diff
@@ -98,32 +95,35 @@ def __str__(self):
...
+        of tensors, where a chain is simply a linked-list of tensors linked via
+        their .child attributes), this method will request that the tensor or 
+        chain being pointed to be serialized and returned from this function.
+        This will typically mean that the remote object will be removed or
+        destroyed. If you merely wish to bring a copy back to the local worker,
+        call .copy() before calling .get().
+        TODO: add param get_copy which doesn't destroy remote if true.
+
+        Args:
+            deregister_ptr (bool): Optional, this determines whether to
```
Comment (mari-linhares) [inline syft/frameworks/torch/tensors/pointer.py]: Same.

File: syft/frameworks/torch/torch_attributes.py [hunk 7]
```diff
@@ -95,15 +96,14 @@ def __init__(self, torch: ModuleType, hook: ModuleType) -> None:
...
-        Check that a command is in a given torch_domain and can be safely used
+        """Checks command is in a given torch_domain and can be safely used
 
         Args:
-            command (str): the command name
-            torch_domain (str): name of the torch domain or module in which the command is
-                supposed to be
-            get_native (boolean): if False (default), return the command name. If True, return
-                the native command function
+            command (str): The command name
```
Comment (mari-linhares) [inline syft/frameworks/torch/torch_attributes.py]: Same.

File: syft/frameworks/torch/torch_attributes.py [hunk 8]
```diff
@@ -115,12 +115,14 @@ def _command_guard(
...
     def _is_command_valid_guard(self, command: str, torch_domain: str) -> bool:
-        """
+        """Validates the command.
+
         Indicates whether a command is valid with respect to the torch guard
 
         Args:
-            command (str): the command to test
-            torch_domain (str): the torch domain or module in which the command is supposed to be
+            command (str): The command to test
```
Comment (mari-linhares) [inline syft/frameworks/torch/torch_attributes.py]: Same.

File: syft/frameworks/torch/torch_attributes.py [hunk 9]
```diff
@@ -146,14 +151,13 @@ def eval_torch_modules(self) -> None:
 
     @staticmethod
     def get_native_torch_name(attr: str) -> str:
-        """
-        Return the name of the native command given the name of the hooked command.
+        """Returns the name of the native command for the given hooked command.
 
         Args:
-            attr (str): the name of the hooked command (ex: torch.add)
+            attr (str): The name of the hooked command (ex: torch.add)
```
Comment (mari-linhares) [inline syft/frameworks/torch/torch_attributes.py]: Same.

**Comment by mari-linhares** [comment]
Thanks for the PR, great work!

File: syft/frameworks/torch/tensors/pointer.py [hunk 10]
```diff
@@ -2,19 +2,21 @@
...
+    PointerTensor has a tensor located somewhere that it points to (they should
+    never exist by themselves). Note that PointerTensor objects can point to
+    both torch.Tensor objects AND to other PointerTensor objects. Furthermore,
+    the objects being pointed to can be on the same machine or (more commonly)
+    on a different one. Note further that a PointerTensor does not know the
+    nature how it sends messages to the tensor it points to (whether over
+    socket, http, or some other protocol) as that functionality is abstracted
+    in the BaseWorker object in self.location.
 
     :Example:
```
Comment (antonrd) [inline syft/frameworks/torch/tensors/pointer.py]: Following other PRs on this issue should these `:Example:` sections be turned into something like `Example:` (indented 4 spaces). I'm looking at PR #1798 and was wondering what's a good way to add these examples in the comment. Not sure the google style guide talks about this in particular.

Comment (katnoria) [reply syft/frameworks/torch/tensors/pointer.py]: Agree, its changed to Example: (indented 4 spaces). I am too not sure what is the best way to add examples in the docstring. Although, Example: (indented 4 spaces) looks readable and more in line with others such as Args, Returns etc.

File: syft/frameworks/torch/tensors/abstract.py (hunk 1 above)
Comment (iamtrask) [reply syft/frameworks/torch/tensors/abstract.py]: Perhaps something like.

"Because PyTorch does not (yet) support functionality for creating arbitrary Tensor types (via subclassing torch.Tensor), in order for our new tensor types (such as PointerTensor) to be usable by the rest of PyTorch (such as PyTorch's layers and loss functions), we need to wrap all of our new tensor types inside of a native PyTorch type. This function adds a .wrap() function to all of our tensor types (by adding it to AbstractTensor), such that (on any custom tensor my_tensor), my_tensor.wrap() will return a tensor that is compatible with the rest of the PyTorch API.

**Comment by iamtrask** [comment]
Welcome to OpenMined, @katnoria!! You should also join our slack team (slack.openmined.org).

File: syft/frameworks/torch/tensors/abstract.py (hunk 1 above)
Comment (katnoria) [reply syft/frameworks/torch/tensors/abstract.py]: Thanks Andrew. I think it provides the useful summary.I have added it to the wrap method.

**Comment by katnoria** [comment]
> Welcome to OpenMined, @katnoria!! You should also join our slack team (slack.openmined.org), particularly the #team_pysyft channel.

Thank you @iamtrask :smile:

File: syft/frameworks/torch/hook.py [hunk 11]
```diff
@@ -90,12 +87,12 @@ def __init__(
         syft.torch = TorchAttributes(torch, self)
 
         if self.local_worker is None:
-            """
-            Every TorchHook instance should have a local worker which is responsible for
```
Comment (robert-wagner) [inline syft/frameworks/torch/hook.py]: This should be formatting as a string comment since it is multiple lines

Comment (katnoria) [reply syft/frameworks/torch/hook.py]: I am a bit confused, the changes I commited have these as "#comment" rather than """strings.

Comment (robert-wagner) [reply syft/frameworks/torch/hook.py]: What I was trying to say was that the """ string was the correct way to format it for consistency

Comment (katnoria) [reply syft/frameworks/torch/hook.py]: Got it, However google style doc shows multiline comments using #.
https://github.com/google/styleguide/blob/gh-pages/pyguide.md#385-block-and-inline-comments

**Comment by alok** [comment]
Late, but you might find this useful: https://github.com/dadadel/pyment
